.github/workflows/build.yml

```diff
@@ -42,3 +45,10 @@ jobs:
       uses: actions/upload-artifact@v2
       with:
         path: ./Artifacts/*
+        
+    - name: Push to Nuget
+      run: dotnet nuget push ./artifacts/*.nupkg -k {ApiKey} --skip-duplicate --no-symbols true
+      if: contains(github.ref, 'refs/tags/')
```

Comment:
Do you think that this will really work?
I'm missing the trigger to build tags.

Reply:
You might be on to something.